web/src/beta/lib/reearth-ui/components/Button/index.tsx

```diff
@@ -67,7 +67,7 @@ const StyledButton = styled("button")<{
   padding:
     size === "small"
       ? iconButton
-        ? `${theme.spacing.micro}px`
+        ? "2px"
```

Comment:
Why do we need to replace theme.spacing.micro with `2`? I think they're same value?